.github/workflows/devhub-ci-test.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
               run: |
                   touch .env.production
                   echo "GATSBY_SITE=devhub" > .env.production; \
-                  echo "GATSBY_PARSER_USER=sophstad" >> .env.production; \
+                  echo "GATSBY_PARSER_USER=jordanstapinski" >> .env.production; \
```

Comment:
could we have a devhub user? or is it a personal account only?

Reply:
I can look into this, it by default uses the account name of your Mac (`echo $USER`). It may be difficult to change. This is how the data is also represented on Atlas

If my Mac had the accountname `devhub` than all would be well :)

Reply:
if only :)